modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -2080,15 +2080,16 @@ void ONNXImporter::parseBatchNormalization(LayerParams& layerParams, const openc
     addLayer(layerParams, node_proto);
 }
 
+// A * B + C = Y, we require that the dimension of A is [m, k], and the dimension of B is [n, k].
+// And the dim of output Y is [m, n]
 void ONNXImporter::parseGemm(LayerParams& layerParams, const opencv_onnx::NodeProto& node_proto)
 {
     CV_Assert(node_proto.input_size() >= 2);
     layerParams.type = "InnerProduct";
     Mat weights = getBlob(node_proto, 1);
-    int ind_num_out = 0;
+
     if (layerParams.has("transB") && !layerParams.get<int>("transB")) {
```

Comment:
Sometimes onnx operator will not set attr, for example:
```
onnx.helper.make_node(
            'Gemm',
            ['a', 'b', 'c'],
            ['y'],
            #transB=0)
```
Both transB and transA are not specified, in case like this, onnx would treat attrs as default value(which is 0),

Reply:
Thanks for mentioning this, it has been fixed.